includes/class-wporg-customizations.php

```diff
@@ -43,6 +43,9 @@ function ( $options, $locale_slug ) {
 				10,
 				2
 			);
+
+			add_filter( 'jetpack_mentions_should_load_ui', '__return_true' );
```

Comment:
If we add this filter then the mentions are also linked:
```suggestion
			add_filter( 'jetpack_mentions_should_load_ui', '__return_true' );
			add_filter( 'jetpack_mentions_allowed_post_types', function( $post_types ) {
				$post_types[] = Helper_Translation_Discussion::POST_TYPE;
				return $post_types;
			} );

```